addons/website_sale/views/templates.xml

```diff
@@ -1009,7 +1009,8 @@
         <t t-call="website.layout">
             <t t-set="additional_title" t-value="product.name" />
             <div itemscope="itemscope" itemtype="http://schema.org/Product" id="wrap" class="js_sale o_wsale_product_page">
-                <div class="oe_structure oe_empty oe_structure_not_nearest" id="oe_structure_website_sale_product_1" data-editor-message="DROP BUILDING BLOCKS HERE TO MAKE THEM AVAILABLE ACROSS ALL PRODUCTS"/>
+                <t t-set="editor_message">DROP BUILDING BLOCKS HERE TO MAKE THEM AVAILABLE ACROSS ALL PRODUCTS</t>
+                <div class="oe_structure oe_empty oe_structure_not_nearest" id="oe_structure_website_sale_product_1" t-att-data-editor-message="editor_message"/>
```

Comment:
Users actually are already aware for the area at the below of your screenshot, because the message was initially shown. It is not shown anymore in your screenshot because the area already has something inside it (you can see the top of a text - image block).

You can see in this screenshot that the same area has a "DRAG BUILDING BLOCK HERE" by default when empty: 
![image](https://github.com/odoo/odoo/assets/30048408/d251b597-8121-42f7-ac67-991636e7c4fb)

It's how those works, once they are filled with something, we don't show the message anymore, to save space and not "annoy" the user with this message every time.
But maybe it could be something to re-think but independant from this PR.

Also, about the "DRAG BUILDING BLOCK HERE" which does not say it's product-specific, it's how it is designed: if we don't mention anything, then it's supposed to act as one would think: it's only linked to the edited record.
So, unless the system hints otherwise, it should be doing what you would expect.